packages/react-reconciler/src/ReactFiberCompleteWork.new.js

```diff
@@ -918,6 +926,11 @@ function completeWork(
       }
       updateHostContainer(current, workInProgress);
       bubbleProperties(workInProgress);
+      if (enableTransitionTracing) {
+        if ((workInProgress.subtreeFlags & Visibility) !== NoFlags) {
+          workInProgress.flags |= Passive;
```

Comment:
What's this for?

Reply:
If a suspense boundary toggles in visibility, we should also schedule a passive effect even if there are no new transitions because this might mean we completed the interaction (or the pending boundaries array changes at least)

Reply:
Ok makes sense, thanks for the added inline comment!